projects/ngrx.io/content/guide/eslint-plugin/install.md

```diff
@@ -0,0 +1,44 @@
+# Installation
+
+## Installing with `npm`
+
+For more information on using `npm` check out the docs <a href="https://docs.npmjs.com/cli/install" target="_blank">here</a>.
+
+```sh
+npm install @ngrx/eslint-plugin --save
+```
+
+## Installing with `yarn`
+
+For more information on using `yarn` check out the docs <a href="https://yarnpkg.com/getting-started/usage#installing-all-the-dependencies" target="_blank">here</a>.
+
+```sh
+yarn add @ngrx/eslin-plugin
+```
+
+## Installing with `ng add`
+
+If your project is using the Angular CLI 6+ then you can install the plugin to your project with the following `ng add` command <a href="https://angular.io/cli/add" target="_blank">(details here)</a>:
+
+```sh
+ng add @ngrx/eslint-plugin
+```
+
+The command will prompt you to select which config you would like to have preconfigured:
+
+| Name                                | Description                                                                                                                       |
```

Comment:
The whole list of configs can be left out imho.
We'll probably forget about it when change them.

Reply:
Do you think we should completely remove the table and just mention it asks for configs?

Reply:
the list of configs is already on the [eslint-plugin/overview](https://github.com/ngrx/platform/blob/master/projects/ngrx.io/content/guide/eslint-plugin/overview.md#configurations) page 👀

Reply:
You can reference the page @markostanimirovic linked.

Reply:
Okay, will do